src/views/collections/CollectionsExplore/CollectionsExplore.vue

```diff
@@ -0,0 +1,212 @@
+  import PuzzleCard from '@/components/Cards/PuzzleCard.vue';
+  import CollectionCard from '@/components/Cards/CollectionCard.vue';
+  import Carousel from '@/components/Common/Carousel.vue';
+  import Pagination from '@/components/PageLayout/Pagination.vue';
+  import Preloader from '@/components/PageLayout/Preloader.vue';
+  import SearchPanel from '@/components/Sidebar/SearchPanel.vue';
+  import {
+    CreatedCollection,
+    CollectionItem,
+    CollectionList,
+    RoadmapAchievement,
+    ProcessedRoadmapAchievement,
+    PuzzleItem,
+  } from '@/types/common-types';
+  import FetchMixin from '@/mixins/FetchMixin';
+  import QuestActivity from '@/views/home/PlayerHome/components/activities/QuestActivity.vue';
+  import TutorialActivity from '@/views/home/PlayerHome/components/activities/TutorialActivity.vue';
+
+  const INITIAL_SORT = 'date';
+  const INITIAL_NUMBER = 18;
+
+  interface CollectionExploreParams {
+    collection_type: string;
+    single: string;
+    joined: string;
+    sort: string;
+    search: string;
+    size: string;
+    uid: number | null;
+  }
+
+  @Component({
+    components: {
+      EternaPage,
+      TagsPanel,
+      FiltersPanel,
+      PuzzleCard,
+      CollectionCard,
+      Swiper,
+      SwiperSlide,
+      BIconChevronRight,
+      BIconChevronLeft,
+      Carousel,
+      Pagination,
+      Preloader,
+      SearchPanel,
+      QuestActivity,
+      TutorialActivity,
+    },
+  })
+  export default class CollectionsExplore extends Mixins(FetchMixin) {
+    collections: CollectionItem[] = [];
+
+    created: CreatedCollection[] = [];
+
+    tenToolsAchievements: RoadmapAchievement[] = [];
+
+    eternaEssentialsAchievements: RoadmapAchievement[] = [];
+
+    masteringEternaAchievements: ProcessedRoadmapAchievement[] = [];
+
+    cleared: PuzzleItem[] = [];
+
+    async fetch() {
+      const { filters, sort, search, size } = this.$route.query;
+      const params = {
+        sort: sort || INITIAL_SORT,
+        size: size || INITIAL_NUMBER,
+        search,
+      } as CollectionExploreParams;
+
+      const ROUTE: string = '/get/?type=collections';
+
+      if (this.$vxm.user.loggedIn) params.uid = this.$vxm.user.uid;
+
+      const res = await Promise.all([
+        this.$http.get(ROUTE, { params,}),
+        this.$http.get('/get/?type=side_project_roadmap'),
+        this.$http.get('/get/?type=puzzle_of_the_week'),
+        this.$http.get('/get/?type=puzzles')
+      ]);
+      
+      this.collections = res[0].data.data.collections;
+      this.created = res[0].data.data.created || [];
+
+      const roadmap = res[1].data.data
+        .achievement_roadmap as RoadmapAchievement[];
+      this.tenToolsAchievements = roadmap.filter((p) => p.key === 'ten_tools');
+      this.eternaEssentialsAchievements = roadmap.filter(
+        (p) => p.key === 'eterna_essentials',
+      );
+      this.masteringEternaAchievements = roadmap
```

Comment:
We probably need to do something different here. Not all of these will be collections (eg, "create a puzzle", "create a tutorial", etc). Maybe we need a new flag in the achievements.json to specify whether or not the achievement is related to a featured collection?

Reply:
I can do that. Are there any achievements that should be excluded other than the puzzle, tutorial, and script achievements?

Reply:
There's a bunch I think - probably easier to consider which are actually related to collections:
- side_quest_eterna3d_expo
- side_quest_eterna100
- side_quest_covid19_potd
- puzzle_of_the_week
- side_quest_best_lab_practices (technically this isn't a static set of puzzles, but I think it probably should be?)

Keep in mind we will need to actually make collections for these, the eterna essentials, and ten tools, and we'll need to change the code related to these (achievement checks, next puzzle feature, pretty much anything else that references the coauthor field in the backend, quest pages on web and mobile, etc) to actually pull from this. The collections will of course need to be created first before we can merge those updates. Also worth noting is that we'll need to figure out how to reference collections in a way that is stable on both dev and prod, as they'll have different NIDs (unless we re-clone the prod DB to dev after creating them)...